.idea/jsLibraryMappings.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Can you update the ".gitignore" file to cause IDE-specific files like this to be ignored and remove this file from the pull request?